source/faq.txt

```diff
@@ -183,6 +183,18 @@ The error message consists of multiple parts:
      can use tools like ``openssl s_client`` to debug TLS/SSL-related
      certificate problems.
 
+.. _csharp-faq-linq-vs-builder:
+
+Should I Use LINQ or Builder Classes When Querying for Documents?
+-----------------------------------------------------------------
+
+If you're used to programming in C#, consider using LINQ because of its similar feel
+to programming in native C#. If you have prior experience with other MongoDB drivers,
+consider using Builder classes because of their consistency with other drivers.
+
+We encourage experimenting with both approaches to determine the most suitable framework
```

Comment:
Suggestion: use something other than "framework" here since they aren't technically frameworks (I'm not set on mechanism either but it was the best word I could think of at the moment)

```suggestion
We encourage experimenting with both approaches to determine the most suitable mechanism
```